### PR TITLE
New change

### PR DESCRIPTION
**Pull request by surendram1234**

### CODE REVIEW

File: dev/components/home/HomePage.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 class HomePage extends React.Component {
     render() {
         return (<div>
-            Home Page
+            Home Page same2233
```
Comment (singhsheetal) [inline dev/components/home/HomePage.js:6]: Please add proper text

File: dev/components/home/HomePage.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
...
        </div>
```
Comment (singhsheetal) [inline dev/components/home/HomePage.js:7]: Change div

**Review by singhsheetal — COMMENTED** [review]
Please do the change

**Comment by singhsheetal** [comment]
Need changes

**Comment by singhsheetal** [comment]
Need chabnges

**Comment by singhsheetal** [comment]
Update change

File: dev/components/home/HomePage.js (hunk 2 above)
Comment (surendram1234) [inline dev/components/home/HomePage.js:7]: Changes done

**Review by surendram1234 — COMMENTED** [review]
Changes donw

**Comment by surendram1234** [comment]
dsdsd

**Pull request closed without merging**